tox.ini

```diff
@@ -79,14 +82,16 @@ extras =
 commands =
     pip freeze --all --no-input
     oldestdeps: python -c "import astropy.time; astropy.time.update_leap_seconds()"
-    !online-!hypothesis-!figure: {env:PYTEST_COMMAND} {posargs}
-    online: {env:PYTEST_COMMAND} --hypothesis-show-statistics --reruns 2 --reruns-delay 15 --timeout=180 --remote-data=any {posargs}
+    !online-!hypothesis-!figure-!vcr: {env:PYTEST_COMMAND} {posargs}
+    online: git clone "https://github.com/dstansby/sunpy-vcr-cassettes" {envtmpdir}/sunpy-vcr-cassettes
```

Comment:
Will we want to use the GitHub Actions cache for `{envtmpdir}/sunpy-vcr-cassettes`? Using the dynamic `{envtmpdir}` might make it more tricky to restore the cache.

Reply:
We could hard code a path that isn't {envtmpdir}. Is the suggestion to cache the whole `sunpy-vcr-cassettes` git repository, and then just run a `git pull` (or something similar) to update it instead of doing a full clone each time? If so I think that's a good idea.

Reply:
I think I'll leave this for now as it's not very trivial. We can alwasy add it later if/when the `sunpy-vcr-cassettes` repo grows in size.

Reply:
I forgot that the cache would have to interact with git. Maybe just leave as-is for now. If we change it to git pull we would also need to have it git clone if the pull fails. We can change the path and add caching later.